### PR TITLE
new module pangolin/updatedata & patch pangolin/run

### PR DESCRIPTION
**Pull request by Joon-Klaps**
## PR checklist

Creates a new module for pangolin for the sub-command `--update-data`.  Moved previous `pangolin` to `pangolin/run` 

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by Joon-Klaps** [comment]
Initially thought of making 2 PR's but because I had to rerun the snapshot I just updated both.

File: modules/nf-core/pangolin/run/tests/main.nf.test [hunk 1]
```diff
@@ -30,6 +33,42 @@ nextflow_process {
...
+                input[1] = PANGOLIN_UPDATEDATA.out.db
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.report[0][1]).name,
```
Comment (famosab) [inline modules/nf-core/pangolin/run/tests/main.nf.test:64]: Is the created report unstable? can we maybe assert a bit more than just the name?

**Review by famosab — APPROVED** [review]
I would assert more than the name but other than that it looks good to me :)

File: modules/nf-core/pangolin/run/tests/main.nf.test (hunk 1 above)
Comment (Joon-Klaps) [reply modules/nf-core/pangolin/run/tests/main.nf.test:64]: Issue is that its not stable as its using a database that gets updated regulary. 
What would you suggest?

Comment (famosab) [reply modules/nf-core/pangolin/run/tests/main.nf.test:64]: maybe check whether the column headers stay the same? but if thats not possible we can also leave it like that

Comment (Joon-Klaps) [reply modules/nf-core/pangolin/run/tests/main.nf.test:64]: ooh yeah totally, didn't think of that! Think with the nft-csv it's easy no.

**Pull request merged**